toqito/nonlocal_games/binary_constraint_system_game.py

```diff
@@ -93,11 +95,12 @@ def generate_solution_group(M: np.ndarray, b: np.ndarray):
 
     Returns
     =======
-    tuple
```

Comment:
[nit] If you look at the style for specifying params and returns elsewhere for functions in toqtio, you'll notice that this style differs a bit from the convention used in the project (which we should be aligning with here).